src/main/java/com/jcabi/beanstalk/maven/plugin/AbstractBeanstalkMojo.java

```diff
@@ -335,23 +314,26 @@ protected boolean validYaml(final String text) {
     }
 
     /**
-     * Validate the war file.
-     *
+     * Verifies that the .ebextensions contains valid configuration file or
+     * files.
      * @param zip Zip file
-     * @throws org.apache.maven.plugin.MojoFailureException Thrown, if
-     *  validation fails.
+     * @throws org.apache.maven.plugin.MojoFailureException Thrown, if the
+     *  .ebextensions does not exist in the WAR file, is empty or one of its
+     *  files is neither valid JSON, nor valid YAML.
      */
     protected void validate(final ZipFile zip) throws MojoFailureException {
-        if (zip.getEntry(".ebextension") == null) {
+        if (zip.getEntry(".ebextensions") == null) {
```

Comment:
This was a bug that became apparent when I moved converted the existing WarFile tests over to use this class.